src/meta/meta_service.cpp

```diff
@@ -76,6 +77,8 @@ meta_service::meta_service()
         "replica server disconnect count in the recent period");
     _unalive_nodes_count.init_app_counter(
         "eon.meta_service", "unalive_nodes", COUNTER_TYPE_NUMBER, "current count of unalive nodes");
+
+    _access_controller = security::create_access_controller(true);
```

Comment:
The parameter is always true?

Reply:
In this pr, it is always true. but it will be false when we want to create a replica_access_controller, which will be added in next pr.